microsetta_interface/templates/new_results_page.jinja2

```diff
@@ -494,9 +498,13 @@
             let layout = {
                 "title": "Ranked Relative Abundance",
                 "yaxis": {
+                    "title": "Common Genera",
                     "automargin": true,
                     "categoryorder": "array",
                     "categoryarray": topXOrdered
+                },
+                xaxis: {
+                    title: 'Rank'
```

Comment:
"<-- less abundant    |     more abundant -->"

(unless that looks totally ugly)